mmv1/products/metastore/api.yaml

```diff
@@ -0,0 +1,235 @@
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: DataprocMetastore
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://metastore.googleapis.com/v1beta/
+scopes:
+  - https://www.googleapis.com/auth/cloud-identity
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Dataproc Metastore API
+    url: https://console.cloud.google.com/apis/library/metastore.googleapis.com
+objects:
+  - !ruby/object:Api::Resource
+    name: 'Service'
+    base_url: "projects/{{project}}/locations/{{location}}/services"
+    create_url: 'projects/{{project}}/locations/{{location}}/services?serviceId={{service_id}}'
+    self_link: "projects/{{project}}/locations/{{location}}/services/{{service_id}}"
+    update_verb: :PATCH
+    update_mask: true
+    min_version: beta
+    description: |
+      A managed metastore service that serves metadata queries.
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        path: 'name'
+        base_url: '{{op_id}}'
+        wait_ms: 1000
+        timeouts: !ruby/object:Api::Timeouts
+          insert_minutes: 40
+          update_minutes: 40
+          delete_minutes: 40
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'response'
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'done'
+        complete: true
+        allowed:
+          - true
+          - false
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error'
+        message: 'message'
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: 'serviceId'
+        required: true
+        input: true
+        url_param_only: true
+        description: |
+          The ID of the metastore service. The id must contain only letters (a-z, A-Z), numbers (0-9), underscores (_),
+          and hyphens (-). Cannot begin or end with underscore or hyphen. Must consist of between
+          3 and 63 characters.
+      - !ruby/object:Api::Type::String
+        name: 'location'
+        url_param_only: true
+        input: true
+        default_value: global
+        description: |
+          The  location where the autoscaling policy should reside.
+          The default value is `global`.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        output: true
+        description: |
+          The relative resource name of the metastore service.
+      - !ruby/object:Api::Type::Time
+        name: 'createTime'
+        description: |
+          Time the AccessPolicy was created in UTC.
```

Comment:
AccessPolicy?

Generally we don't include create/update times as they are not particularly useful to users

Reply:
bad copy job, cool, removing these two